src/udf/src/main/scala/UtilityUDFs.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
```

Comment:
Move into a sub-package

Reply:
made a udfs object. It is imported like a package.